Frontend/ui-customer/src/Components/SearchLocationModel/View.purs

```diff
@@ -637,7 +679,7 @@ bottomBtnsView state push =
             ( \idx item ->
                 linearLayout
                   [ height MATCH_PARENT
-                  , width if (state.isSource == Just true && state.isSearchLocation /= LocateOnMap) then (V 190) else MATCH_PARENT
+                  , width if (state.isSource == Just true && not any (_ == state.isSearchLocation) [LocateOnMap, RouteMap]) then (V 190) else MATCH_PARENT
```

Comment:
`(state.isSource == Just true && not any (_ == state.isSearchLocation) [LocateOnMap, RouteMap])` use this get a config and then use wherever required. 

eg - `config.width`